R/write_pxl_file.R

```diff
@@ -663,14 +682,14 @@ WriteMPX_pxl_file <- function (
       return(cg@counts %>% Matrix::t())
     }) %>% do.call(cbind, .)
     colnames(merged_counts) <- 1:ncol(merged_counts)
-    #merged_counts <- SeuratObject::RowMergeSparseMatrices(all_counts[[1]], all_counts[-1])
 
     merged_layouts <- lapply(names(layouts), function(layout_type) {
       layout_tbl <- layouts[[layout_type]]
+      # If the layout only has x,y columns, add a third z column of NAs
```

Comment:
Why is this necessary?

Reply:
If I remember correctly, this is how coordinates are stored in the PXL files. If you compute pmds 2D, you still get a z column with empty values. But I need to double check this with Johan. Unfortunately, I don't have access to any PXL files with 2D layouts.